src/pages/chainlink-nodes/v1/fulfilling-requests.md

```diff
@@ -0,0 +1,159 @@
+---
+layout: ../../../layouts/MainLayout.astro
+section: nodeOperator
+date: Last Modified
+title: "Fulfilling Requests"
+permalink: "docs/fulfilling-requests/"
+whatsnext:
+  {
+    "Performing System Maintenance": "/chainlink-nodes/resources/performing-system-maintenance/",
+    "v2 Jobs": "/chainlink-nodes/oracle-jobs/jobs/",
+    "Security and Operation Best Practices": "/chainlink-nodes/resources/best-security-practices/",
+  }
+metadata:
+  title: "Chainlink Node Operators: Fulfilling Requests"
+  description: "Deploy your own Operator contract and add jobs to your node so that it can provide data to smart contracts."
+setup: |
+  import CodeSample from "@components/CodeSample/CodeSample.astro"
+---
+
+:::note[Run a Chainlink node]
+This guide assumes you have a running Chainlink node. If not, Follow the [Running a Chainlink Node locally](/chainlink-nodes/v1/running-a-chainlink-node) guide.
+:::
+
+Now that your Chainlink node is running , you can use your own node to fulfill requests. This guide will show you how to deploy your own Operator contract and add jobs to your node so that it can provide data to smart contracts.
```

Comment:
```suggestion
You can use your Chainlink nodes to fulfill requests. This guide shows you how to deploy your own operator contract and add jobs to your node so that it can provide data to smart contracts.
```